lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueDocumentEventRepository.java

```diff
@@ -60,20 +59,19 @@
      * or superseded), should be put back in the document event entity pool to be processed later.
      */
     private final int documentEventBatchSize;
-    private final Locking locking;
+    private final LockStrategy lockStrategy;
     private final Map<String, ? extends DocumentEventFactory> documentEventFactoriesByType;
```

Comment:
I actually had a type like this, but Kevin thought this was more confusing that just using a good ol' Map. I agree. We only need to check in one place.